skbeam/core/accumulators/binned_statistic.py

```diff
@@ -434,36 +458,26 @@ def __init__(self, rbins, phibins, rowsize, colsize,
                 will be called on the values in each bin.  Empty bins will be
                 represented by function([]), or NaN if this returns an error.
         """
+        if origin is None:
+            origin = shape[0]//2, shape[1]//2
```

Comment:
@chrisvam @danielballan @tacaswell 

sorry to reference a merged PR, but I forgot to mention this. I think this should rather be:
```python
origin = shape[0]/2., shape[1]/2.
```
the reason being that origin may be a `float`.

Would there be objections to changing this default?